joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/datastructures/GoGlobal.scala

```diff
@@ -25,6 +25,8 @@ object GoGlobal extends Global {
 
   val lambdaSignatureToLambdaTypeMap: ConcurrentHashMap[String, Set[(String, String)]] = new ConcurrentHashMap()
 
+  val pkgLevelVarAndConstantAstMap: ConcurrentHashMap[String, Set[(Ast, String)]] = new ConcurrentHashMap()
```

Comment:
Just a comment, but the Scala version of this is `TrieMap`

Reply:
Let me try that in the next PR.